### PR TITLE
Snap install

### PR DESCRIPTION
**Pull request by phw**
### Summary

This is a…

- [ ] Correction
- [x] Addition
- [ ] Restructuring
- [ ] Minor / simple change (like a typo)
- [ ] Other

### Reason for the Change
The Snap version for Picard has been updated and is now on par with the Flatpak version.

### Description of the Change

Update Linux instructions to provide Snap as an alternative installation method. In general restructure the Linux instructions a bit and also mention distribution packages (which likely still are the most common way to install Picard on Linux).

### Additional Action Required

Linux with its special way of handling software distribution warrants some more detailed descriptions. But In general we could detail the install instructions for macOS and Windows more as well. E.g. for Windows we could describe the individual options (installer, portable, Windows 10 store).

So for the future we might aim for a general Download & Install page, but this then links to platform specific pages with more details.

### CODE REVIEW

File: _locale/fr/LC_MESSAGES/getting_started/download.po [hunk 1]
```diff
@@ -22,91 +22,145 @@ msgstr ""
...
+"Picard à partir du centre logiciel de votre distribution, par ex. Logiciel "
+"Ubuntu ou KDE Discover. Vous pouvez également installer Picard à partir de la "
+"ligne de commande:"
+
+#: ../../getting_started/download.rst:61
+msgid ""
+":index:`Installing from your distribution's package repository <pair: "
+"install; Linux package>`"
+msgstr ""
+": index: `Installation à partir du référentiel de packages de votre "
```
Comment (rdswift) [inline _locale/fr/LC_MESSAGES/getting_started/download.po:137]: Need to remove the space between the first colon and 'index', and the space between the second colon and the backtick.

**Review by rdswift — CHANGES_REQUESTED** [review]
This looks good, and you even updated the translation file.  Thank-you!

File: _locale/fr/LC_MESSAGES/getting_started/download.po [hunk 2]
```diff
@@ -22,91 +22,145 @@ msgstr ""
...
+"distribution <pair: install; Paquet Linux> `"
```
Comment (rdswift) [inline _locale/fr/LC_MESSAGES/getting_started/download.po:138]: Need to remove extra spaces.  Every time I use Google Translate it changes all the spacing (and sometimes the backticks) around the restructured-text directives.

```suggestion
":index:`Installation à partir du référentiel de packages de votre "
"distribution <pair: install; Paquet Linux>`"
```

**Review by rdswift — CHANGES_REQUESTED** [review]

File: _locale/fr/LC_MESSAGES/getting_started/download.po (hunk 2 above)
Comment (phw) [reply _locale/fr/LC_MESSAGES/getting_started/download.po]: Oh yes, good catch. Was hard to spot locally (using poedit). One needs to be careful with the Google translate results

**Pull request merged**